app/controllers/sessions.rb

```diff
@@ -0,0 +1,21 @@
+get '/sessions/new' do
+  erb :'sessions/new'
+end
+
+post '/sessions' do
+  puts params
+  @user = User.authenticate(params[:username], params[:password])
+  if @user
+    session[:user_id] = @user.id
+    "login success"
+    redirect "/suggestions/#{@user.id}"
```

Comment:
Heads up: this redirect is to suggestions but with the _user_ ID.  I am guessing this is not going to do what you are aiming to do.